src/stream.rs

```diff
@@ -106,7 +106,7 @@ pub struct Utf8LossyDecoder<Sink, A=NonAtomic>
           A: Atomicity
 {
     decoder: utf8::Decoder,
-    sink: Sink,
+    pub inner_sink: Sink,
```

Comment:
Given that you've written the `inner_sink` and `inner_sink_mut` function below, is it still necessary to make this field public?